src/main.js

```diff
@@ -43,23 +20,24 @@ Object.assign(extension, {
    * @return {Element}
    */
   getElementToActivate(results, linkOnly = false) {
-    const focusedElement = document.activeElement;
+        const focusedElement = document.activeElement;
 
-    if (focusedElement !== null) {
-      const isLink = (focusedElement.localName === 'a' && focusedElement.hasAttribute('href'));
+        if (focusedElement !== null) {
+          const isLink = (focusedElement.localName === 'a' && focusedElement.hasAttribute('href'));
 
-      if (!linkOnly || isLink) {
-        return focusedElement;
-      }
-    }
+          if (!linkOnly || isLink) {
+            return focusedElement;
+          }
+        }
 
-    return results.items[results.focusedIndex].anchor;
+        return results.items[results.focusedIndex].anchor;
   },
 
-  initResultsNavigation() {
+  initResultsNavigation(searchEngine) {
```

Comment:
Shouldn't this function use `this.searchEngine` instead of getting it as a parameter?

Reply:
Yes, it should. See 3f4d28f